packages/@o3r/styling/builders/style-extractor/schema.ts

```diff
@@ -15,4 +15,7 @@ export interface StyleExtractorBuilderSchema extends JsonObject {
 
   /** List of libraries imported */
   libraries: string[];
+
+  /** Library/Application name to be assigned into metadata */
+  name: string;
```

Comment:
As it is not mandatory, please specify `| null` to make sure to handle the nullish case